sdk/cognitiveservices/CognitiveServices.AnomalyDetector/AnomalyDetector.sln

```diff
@@ -0,0 +1,33 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I was expecting that you would also remove "CognitiveServices" from the beginning of all these directory names.